### DISCUSSION TITLE
All my wallet tokens have disappeared.

### DISCUSSION THREAD

**Discussion by smarcos23** [iotaledger/firefly#7314] [Help & Support]
All my wallet tokens have disappeared. There is no transaction history in or out. I have already updated everything, what can this be due to?

**Comment by begonaalvarezd** [comment]
Hi, first of all make sure you are connected to mainnet and not tested/devnet
If you are connected to mainnet and your funds are still not visible, please go to settings > advanced > balance/wallet finder and run a couple of searches to look for your funds.

**Reply by JTGBonheur** [reply]
Und wenn das nicht hilft ?

**Reply by begonaalvarezd** [reply]
can you explain a bit more? did you try what I mentioned above?

**Comment by parkseungho7** [comment]
지갑을 2개로 분리해두었는데 갑자기 지갑1개가 보이지 않습니다.

**Reply by begonaalvarezd** [reply]
same as the comment above, please run settings > advanced > balance finder to discover more wallets

**Comment by JTGBonheur** [comment]
Ich hoffe es ist ok wenn ich auf deutsch schreibe, mein Englisch ist ein bisschen eingerostet.
Also das mit dem suchen meiner Wallet Bestände oder Guthaben habe ich gemacht.
Ich habe auch versucht mein letztes Backup wieder herzustellen aber leider hab ich keine Ahnung wie das geht.

Am 29.02.2024 um 15:18 schrieb Begoña Álvarez de la Cruz ***@***.***>:

﻿

can you explain a bit more? did you try what I mentioned above?

**Reply by begonaalvarezd** [reply]
I suggest you visit our discord https://discord.iota.org/ and open a help ticket there, we have community members that speak german and could help you in german if you prefer. You will get a better assistance there too, github is for bugs not for support tickets. You can find me there too and I can try helping you. Im `begonaalvarez` in discord 🌷

**Comment by JTGBonheur** [comment]
I don’t know why, but it is kicking me out by joining the discord channel. Maybe I’m stupid.
I followed your link and when it says please verify you account it throws me out. Damned. What should I do now ?

Am 29.02.2024 um 18:51 schrieb Begoña Álvarez de la Cruz ***@***.***>:

﻿

I suggest you visit our discord https://discord.iota.org/ and open a help ticket there, we have community members that speak german and could help you in german if you prefer. You will get a better assistance there too, github is for bugs not for support tickets. You can find me there too and I can try helping you. Im begonaalvarez in discord 🌷

**Comment by JTGBonheur** [comment]
Hey there, can you help me please 🙏

Regarding
Janosch Glück

Am 01.03.2024 um 15:10 schrieb Janosch Glück ***@***.***>:

﻿ I don’t know why, but it is kicking me out by joining the discord channel. Maybe I’m stupid.
I followed your link and when it says please verify you account it throws me out. Damned. What should I do now ?

Am 29.02.2024 um 18:51 schrieb Begoña Álvarez de la Cruz ***@***.***>:

﻿

I suggest you visit our discord https://discord.iota.org/ and open a help ticket there, we have community members that speak german and could help you in german if you prefer. You will get a better assistance there too, github is for bugs not for support tickets. You can find me there too and I can try helping you. Im begonaalvarez in discord 🌷
